src/elm/Page/Community/Settings/Settings.elm

```diff
@@ -0,0 +1,149 @@
+module Page.Community.Settings.Settings exposing (Model, Msg, init, msgToString, update, view)
+
+import Api.Graphql
+import Community
+import Eos exposing (Symbol)
+import Graphql.Http
+import Html exposing (Html, a, button, div, span, text)
+import Html.Attributes exposing (class, style)
+import I18Next exposing (t)
+import Page exposing (viewHeader)
+import Route exposing (Route)
+import Session.LoggedIn as LoggedIn exposing (External(..))
+import Session.Shared exposing (Shared)
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> Symbol -> ( Model, Cmd Msg )
+init { shared } symbol =
+    ( initModel symbol
+    , Api.Graphql.query shared (Community.communityQuery symbol) CompletedLoad
+    )
+
+
+type alias Model =
+    { currency : String
+    , status : Status
+    }
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+type Status
+    = Loading
+    | LoadingFailed (Graphql.Http.Error (Maybe Community.Model))
+    | Loaded Community.Model
+
+
+type Msg
+    = CompletedLoad (Result (Graphql.Http.Error (Maybe Community.Model)) (Maybe Community.Model))
+
+
+initModel : Symbol -> Model
+initModel symbol =
+    { currency = Eos.symbolToString symbol
+    , status = Loading
+    }
+
+
+view : LoggedIn.Model -> Model -> { title : String, content : Html Msg }
+view loggedIn model =
+    let
+        title =
+            "Settings"
+
+        content =
+            let
+                translate =
+                    t loggedIn.shared.translations
+
+                headerText =
+                    translate "community.edit.title"
+
+                header =
+                    Page.viewHeader loggedIn headerText Route.Dashboard
+            in
+            case model.status of
+                Loading ->
+                    div []
+                        [ header
+                        , div [ class "container mx-auto px-4" ]
+                            [ Page.fullPageLoading ]
+                        ]
+
+                Loaded community ->
+                    div []
+                        [ header
+                        , viewSettingsList loggedIn.shared community.symbol
+                        ]
+
+                LoadingFailed e ->
+                    Page.fullPageGraphQLError headerText e
+    in
+    { title = title
+    , content = content
+    }
+
+
+viewSettingsList : Shared -> Symbol -> Html Msg
+viewSettingsList shared symbol =
+    let
+        translate =
+            t shared.translations
+    in
+    div
+        [ class "grid my-4"
+        , style "grid-template-columns" "0 1fr 0"
+        , style "grid-template-rows" "auto"
+        , style "grid-gap" "16px"
+        ]
+        [ settingCard (translate "settings.community_info.title") (translate "settings.community_info.description") Route.Dashboard
```

Comment:
I'm confused about this, we don't still have some of those screens, and all of them are just mocked to send us back to dashboard. We can add it later, but it don't make sense to push something into production if it doesn't work, seems misleading.

Reply:
Fair enough. Would it be better to comment them out, or delete entirely?

Reply:
we can delete it since it will be available at the new settings page

Reply:
you made it in a way that it’s easy to add back if needed (: i would say to just remove it